install_files/ansible-base/roles/restrict-direct-access/templates/sshd_config

```diff
@@ -22,20 +33,34 @@ ChallengeResponseAuthentication no
 KerberosAuthentication no
 KerberosGetAFSToken no
 GSSAPIAuthentication no
-X11Forwarding no
-X11DisplayOffset 10
-PrintMotd no
-PrintLastLog yes
-TCPKeepAlive yes
-AcceptEnv LANG LC_*
 UsePAM no
 UseDNS no
+
+# Cipher selection
+
+Ciphers chacha20-poly1305@openssh.com,aes256-gcm@openssh.com,aes128-gcm@openssh.com,aes256-ctr,aes128-ctr
+HostKeyAlgorithms ssh-ed25519,rsa-sha2-256,rsa-sha2-512
```

Comment:
It looks like this ordering forces a new host key to be generated when the new config is applied.  Since ECDSA is being removed altogether I don't see that there's a way around this in Xenial. It might make sense to have `tailsconfig` remove the old host key entries on the workstation if found, and alert the user that they can expect to see prompts on their first SSH/playbook run.